apps/marketplace/components/shared/form/FileInput.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import isEmpty from 'lodash/isEmpty'
+import get from 'lodash/get'
+
+import styles from './FilesInput.scss'
+
+class FileInput extends React.Component {
+  constructor() {
+    super()
+    this.onReset = this.onReset.bind(this)
```

Comment:
You don't need to bind functions in the constructor if you use es6 arrow function syntax. That autobinds the function to this.